hvplot/converter.py

```diff
@@ -1172,6 +1172,10 @@ def _category_plot(self, element, x, y, data):
                 .relabel(**self._relabel).opts(**opts))
 
     def bar(self, x, y, data=None):
+        # As of bokeh 1.3.0 and holoviews 1.12.4, width is valid on style but throws
+        # a param warning. Use bar_width to control width of bars.
+        self._style_opts.pop('width', None)
```

Comment:
Why is width in the style opts?

Reply:
It is a valid bokeh style, so when the kwds get passed through the validator it is added to style_opts.

Reply:
But surely it should get captured and dropped during plot option handling before the style opts pick it up?

Reply:
Hmm I think with the opts, maybe the plot_opts should be popped before passing all the kwds to style validator.

Reply:
Right, only unprocessed kwds should ever making through to the style validator.